config.go

```diff
@@ -700,6 +700,17 @@ type Config struct {
 	//
 	// Required: defaults to false
 	SkipTargetVerification bool
+	//
+	// The Ferry will enforce that the source DB has no Foreign Key constraints.
+	// It will error out before running.
+	//
+	// It can be overriden to skip this enforcement.
+	//
+	// This allows for custom implementations to circumvent
```

Comment:
what do you mean by custom implementations?

Reply:
I realize now, the wording is quite ambiguous. I meant that a ferry could employ its own table ordering to "work around" foreign key constraints. I'll make some edits along with your suggestions below 👍